### PR TITLE
added i am here and a link to your current schedule

### PR DESCRIPTION
**Pull request by ava-silver**
two useful links that are neat

also added semicolons in `links.js`

### CODE REVIEW

**Comment by breqdev** [comment]
https://github.com/breqdev/nuisance/blob/04059b010f4e2edc662dd4dbf345cac5cb012617/.prettierrc.json#L3

File: src/links.js [hunk 1]
```diff
@@ -85,6 +89,16 @@ const LINKS = [
             },
         ],
     },
+    {
+        title: "i am here",
+        description: "complete i am here",
```
Comment (breqdev) [inline src/links.js]: consider thinking of a more catchy description

**Review by breqdev — CHANGES_REQUESTED** [review]

**Comment by ava-silver** [comment]
> https://github.com/breqdev/nuisance/blob/04059b010f4e2edc662dd4dbf345cac5cb012617/.prettierrc.json#L3

dear god why thats awful

**Review by breqdev — APPROVED** [review]
this PR is a member of the LGTM community 🏳️‍🌈

**Pull request merged**